Cargo.toml

```diff
@@ -13,3 +13,4 @@ directories = "5"
 rustc_version = "0.4"
 rustc-build-sysroot = "0.4.0"
 serde_json = "1.0.87"
+apple-sdk = { version = "0.5.1", default-features = false }
```

Comment:
This is a strange dependency. https://crates.io/crates/apple-sdk links to https://github.com/indygreg/toolchain-tools/ as the source repository but I cannot find the sources there. I also cannot find major crates using this dependency.

In its current form I am not comfortable adding this dependency. Can you say more about why it is required?

Reply:
Ah I found the sources: https://github.com/indygreg/apple-platform-rs/tree/main/apple-sdk/

Reply:
Yeah, confused me at first, they moved it over to https://github.com/indygreg/apple-platform-rs - I've opened https://github.com/indygreg/apple-platform-rs/pull/122 to fix that.

---

It's used only once for calling [`DeveloperDirectory::find_default`](https://docs.rs/apple-sdk/latest/apple_sdk/struct.DeveloperDirectory.html#method.find_default), which implements a sane search order for finding the include directory. I think it's roughly the same logic as what clang does, though it's been a while since I looked at that in detail.

I guess `cargo-careful` could also just shell out to `xcode-select --print-path` itself, I just picked the slightly more robust option, but am fine with either.

Reply:
> I guess cargo-careful could also just shell out to xcode-select --print-path itself, I just picked the slightly more robust option, but am fine with either.

If that's all it takes then that seems preferable, yes.

Reply:
Okay, I've implemented that functionality now, and removed the dependency on `apple-sdk`.